ambry-api/src/main/java/com/github/ambry/rest/RestUtils.java

```diff
@@ -291,6 +295,27 @@
     public static final String REQUEST_COST = "x-ambry-request-cost";
   }
 
+  /**
+   * Headers with information about request quota and usage.
+   */
+  public static final class RequestQuotaHeaders {
+
+    /**
+     * Response header indicating user quota usage information.
+     */
+    public static final String USER_QUOTA_USAGE = "x-ambry-user-quota-usage";
+
+    /**
+     * Response header indicating user quota warning.
+     */
+    public static final String USER_QUOTA_WARNING = "x-ambry-user-quota-warning";
```

Comment:
can you rename this to REQUEST_QUOTA_XXX.

Reply:
I don’t want to rename the individual fields, because I want the headers to say `x-ambry-user-quota-usage` instead of `x-ambry-request-quota-usage`.
For users the message should be clear that its about the user quota.